.run/go test all.run.xml

```diff
@@ -1,12 +1,11 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="go test all" type="Multirun" factoryName="Multirun" singleton="false" separateTabs="true" reuseTabsWithFailures="false" startOneByOne="true" markFailedProcess="true" hideSuccessProcess="true" delayTime="4.0">
```

Comment:
Should I revert unrelated changes like these? Probs generated by IntelliJ though

Reply:
Generally yes. But in this case it's unclear whether this is a "correct" update from IntelliJ, so I'm fine with leaving it in and if it gets reverted in a subsequent PR then fine